orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -40,6 +40,7 @@ class CreateWebhookTask implements RetryableTask {
 
   long backoffPeriod = 10000
   long timeout = 300000
+  static final String NOBODYRETURNED = "<No body returned by server>";
```

Comment:
based on convo with @ajordens will remove this in favor of indicating that no body was present in the UI